File: app/src/main/java/com/quran/labs/androidquran/ui/helpers/JumpDestination.java
package com.quran.labs.androidquran.ui.helpers;

/**
 * Activity or fragment implements this is meant to be a jump destination/target.
 */
public interface JumpDestination {
  void jumpTo(int page);

  void jumpToAndHighlight(int page, int sura, int ayah);
}

File: app/src/main/java/com/quran/labs/androidquran/ui/helpers/JumpDestination.kt
package com.quran.labs.androidquran.ui.helpers

/**
 * Activity or fragment implements this is meant to be a jump destination/target.
 */
interface JumpDestination {
  fun jumpTo(page: Int)
  fun jumpToAndHighlight(page: Int, sura: Int, ayah: Int)
}
